assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/calico-host-protection.yaml.tmpl

```diff
@@ -10,6 +10,7 @@ hostEndpoints:
 %{ for h in host_endpoints ~}
 - name: ${h.name}
   interfaceName: ${h.interface_name}
+  nodeName: ${h.node_name}
```

Comment:
I don't follow why we have this duplication here. Is this just for the controllers?

Reply:
This is Terraform template which generates Helm values.yaml file for the chart.  What do you mean by duplication?

Reply:
This file calico-host-protection.yaml.tmpl seems like a template (given the name) and seems very similar to the yaml in the chart.

I see now that this is the "vules.yaml" file for the chart to use, but wasn't obvious from the name before to me.

I'd very much like to see how to simplify this flow in the future

Reply:
https://github.com/kinvolk/lokomotive/issues/181 is one suggestion of how to simplify it